### PR TITLE
Create all the models

### PR DESCRIPTION
**Pull request by khanh-devos**
Create all the models.
Notice: login is not required currently.

image

### CODE REVIEW

File: app/models/reservation.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Reservation < ApplicationRecord
```
Comment (souad988) [inline app/models/reservation.rb:1]: well done @khanh-devos  but I thing this model is supposed to be a many to many relation so we need to link it to a user a reservation belongs to a user and a belongs to a bike, I also think  that the reservation date is missing here

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @khanh-devos , 

You've done well implementing the models 👍🏼 👍🏼 👍🏼  , but there are still some issues to fix before we can merge this part.


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

File: app/models/reservation.rb (hunk 1 above)
Comment (khanh-devos) [reply app/models/reservation.rb:1]: Thanks for your review @souad988 
I think that "reservation_date" means "starting_date" & "ending_date"

**Review by souad988 — APPROVED** [review]
Hi @khanh-devos , 

This feature is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

**Pull request merged**